app/controllers/sections_controller.rb

```diff
@@ -0,0 +1,51 @@
+class SectionsController < ApplicationController
+
+  def index
+    @sections = Section.all
+    @items = MenuItem.all
```

Comment:
Related to my comment about using the `has_many` in the `index` file, that would also allow you to not need to assign a variable here.